src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,44 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
```

Comment:
Casting the result of `injector.getInstance(BookDao.class)` to `BookDao` assumes that the injector is correctly configured to return an instance of `BookDao`. Verify that the injector is set up properly to avoid `ClassCastException`.